### PR TITLE
BI-20: khanhvnv - Implement the simple system for managing user accounts

### PR DESCRIPTION
**Pull request by khanhvnv**
## Exercise

* [Exercise-20](https://github.com/NT-Line-team/Behave_test/issues/20)

## Overview
A system has 2 features that allow user to Register and Login
#20 
Create the above system as simple as possible with the following workflow:

Register account
Login
End
 
### Docs
- [ ] No Doc Changes.

### CODE REVIEW

**Comment by hungphamviet** [comment]
Hi @khanhvnv ,

The description in your PR is only the template. Please fill in all information for it.

File: python_basic/khanhvnv/exercise19.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#1/1/1900 Monday
```
Comment (hungphamviet) [inline python_basic/khanhvnv/exercise19.py]: Please rename the file to what you intend to implement. The name `exercise19.py` is not self described to other people.

File: python_basic/khanhvnv/exercise20.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
+print "Register account"
```
Comment (hungphamviet) [inline python_basic/khanhvnv/exercise20.py]: Same for this file, please rename it.

**Comment by hungphamviet** [comment]
Also, could you please create 2 separate pull requests for 2 exercises? I believe we should manage 1 PR per exercise.

**Comment by khanhvnv** [comment]
to start separate pull request

**Pull request closed without merging**